src/nn/ops/conv2d.ts

```diff
@@ -99,4 +99,4 @@ export class Conv2d extends SingleOutputOperation {
     }
     return output;
   }
-}
\ No newline at end of file
+}
```

Comment:
What's the reason to modify `conv2d.ts`?

Reply:
@huningxin Fixed an bug  of getting conv2d inputChannels for 'nchw' layout. After transposing input tensor of 'nchw' layout to 'nhwc' layout,  the index of inputChannels will be changed from 1 to 3, so I moved the line 
```javascript
inputChannels = input.shape[1];
```
up.

Reply:
good catch. Thanks for the fix!